zpay32/fuzz_test.go

```diff
@@ -39,7 +39,7 @@ func FuzzEncode(f *testing.F) {
 
 		// Initialize the static key we will be using for this fuzz
 		// test.
-		testPrivKeyBytes, _ := hex.DecodeString("e126f68f7eafcc8b74f54d269fe206be715000f94dac067d1c04a8ca3b2db734") // nolint:lll
+		testPrivKeyBytes, _ := hex.DecodeString("e126f68f7eafcc8b74f54d269fe206be715000f94dac067d1c04a8ca3b2db734") //nolint:lll
```

Comment:
nit: wrap code instead?

Reply:
cool let me remove it actually, it's already defined in another file actually